test/conformance/device/urDevicePartition.cpp

```diff
@@ -4,6 +4,15 @@
 
 using urDevicePartitionTest = uur::urAllDevicesTest;
 
+void getNumberComputeUnits(ur_device_handle_t_ *device,
```

Comment:
This was to get ASSERT_SUCCESS and ASSERT_NE to work. If the return type of the function is not void, then it will not compile. The alternative would be to use the EXPECT functions.